src/utils.rs

```diff
@@ -8,10 +8,24 @@ use tilejson::Bounds;
 
 use crate::source::{Query, Xyz};
 
-pub fn prettify_error<E: std::fmt::Display>(message: String) -> impl Fn(E) -> std::io::Error {
-    move |error| std::io::Error::new(std::io::ErrorKind::Other, format!("{message}: {error}"))
+#[macro_export]
+macro_rules! prettify_error {
+    ($error:ident,$info:expr,$($arg:expr),+) => {
```

Comment:
spaces keep readers happy :)  I know `cargo fmt` doesn't try to reformat macros (yet), so we will have to do the heavy lifting.  Unlike other non-important stuff, good format of the source code will keep other developers much happier when they try to understand why nothing works :)

Also, we want to allow users to have a (single) trailing comma, here and in other macro variants.

```suggestion
    ($error:ident, $info:expr, $($arg:expr),+ $(,)?) => {
```